src/blocks.php

```diff
@@ -69,3 +83,45 @@ function stackable_register_blocks() {
 	}
 	add_action( 'init', 'stackable_register_blocks' );
 }
+
+/**
+ * Allow our blocks to display post excerpts
+ * when calling `get_the_excerpt` function.
+ *
+ * @see https://developer.wordpress.org/reference/hooks/excerpt_allowed_blocks/
+ */
+if ( ! function_exists( 'stackable_add_excerpt_wrapper_blocks' ) ) {
+	/**
+	 * Register stackable blocks with inner blocks.
+	 */
+	function stackable_add_excerpt_wrapper_blocks( $allowed_wrapper_blocks ) {
+		global $stk_wrapper_block_folders;
+		$allowed_stackable_wrapper_blocks = array();
+
+		foreach ( $stk_wrapper_block_folders as $stk_wrapper_block ) {
+			array_push( $allowed_stackable_wrapper_blocks, 'stackable/' . $stk_wrapper_block );
```

Comment:
Don't use the folder names directly as the name of the block, these may be different from the actual block name. Maybe you can refactor the code above that gathers the block meta data `$metadata = json_decode( file_get_contents( $block_json_file ), true );` to get the block name `$metadata['name']`